prov/sockets/src/sock_progress.c

```diff
@@ -2534,7 +2536,7 @@ static void sock_pe_wait(struct sock_pe *pe)
 
 	ret = fi_epoll_wait(pe->epoll_set, ep_contexts, 1, -1);
 	if (ret < 0)
-		SOCK_LOG_ERROR("poll failed : %s\n", strerror(errno));
+		SOCK_LOG_ERROR("poll failed : %s\n", strerror(ofi_sockerr()));
```

Comment:
epoll with sockerr

Reply:
Because there is no epoll for Windows (there is IOCPs instead - it would be good idea to implement epoll via IOCPs in the future), this part of code can't be reachable on Windows. Adding that here, I want to show that we should use ofi_sockerr() instead of errno anywhere.
But if we implemented epoll via IOCPs, we should check `GetLastError()` instead of `WSAGetLastError()` here (i.e. ofi_sockerr isn't applicable here).
But we should check ofi_sockerr() in case of poll (poll -> WSAPoll on Windows) - It's a bit confusing for developers. I think I need to drop the changes for epoll and we should come up with an approach that helps up to avoid ambiguity